pyprof/prof/softmax.py

```diff
@@ -86,16 +86,11 @@ def __init__(self, d):
         assert (mod in ["torch", "torch.nn.functional"])
         assert (op == "log_softmax")
 
-        #Filter out named parameters
-        args = list(filter(lambda x: x['name'] == '', args))
```

Comment:
See above.

Reply:
ditto - bad assumption about keyword and non keyword args

Reply:
I can add a comment in the `nvmarker.py` file to more clearly specify what do the individual fields capture.